.bazelci/presubmit.yml

```diff
@@ -2,32 +2,49 @@
 bazel: 0.28.1
 tasks:
   ubuntu1604:
-    name: ":linux:"
+    name: "PlaidML Ubuntu 16.04"
     shell_commands:
-    - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-    - bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
-    build_targets:
-    - "//plaidml/keras:wheel"
-    build_flags:
-    - "--config=linux_x86_64"
+    - curl -O https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh 
+    - sh ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
+    - mkdir neo
+    - cd neo
+    - wget https://github.com/intel/compute-runtime/releases/download/19.35.13977/intel-gmmlib_19.2.3_amd64.deb
+    - wget https://github.com/intel/compute-runtime/releases/download/19.35.13977/intel-igc-core_1.0.11-2500_amd64.deb
+    - wget https://github.com/intel/compute-runtime/releases/download/19.35.13977/intel-igc-opencl_1.0.11-2500_amd64.deb
+    - wget https://github.com/intel/compute-runtime/releases/download/19.35.13977/intel-opencl_19.35.13977_amd64.deb
+    - wget https://github.com/intel/compute-runtime/releases/download/19.35.13977/intel-ocloc_19.35.13977_amd64.deb
+    - dpkg -i *.deb
+    - cd ..
+    environment:
+      PATH: /var/lib/buildkite-agent/miniconda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/go1.12.6.linux-amd64/bin:/opt/swift-4.2.1-RELEASE-ubuntu16.04/usr/bin
+      PLAIDML_DEVICE_IDS: llvm_cpu.0
+      PLAIDML_EXPERIMENTAL: 1
+      PLAIDML_TELEMETRY: 0
+    test_flags:
+    - "--config=linux_x86_64_bazelci"
```

Comment:
Actually, I think we can do this:

```
- "--config=linux_x86_64"
- "--config=bazelci"
```

And then the `bazel.rc` could have: 
```
build:bazelci --build_tag_filters=-darwin,-metal,-msvc,-opencl,-pytorch
test:bazelci --test_tag_filters=-darwin,-metal,-msvc,-opencl,-pytorch
```

Although not sure how bazel handles combining multiple configs.